crypto/Cargo.toml

```diff
@@ -55,15 +55,13 @@ base64.workspace = true
 tracing.workspace = true
 
 [dependencies.proteus-wasm]
-version = "2.1"
+workspace = true
 features = ["hazmat"]
 optional = true
-git = "https://github.com/wireapp/proteus"
-branch = "2.x"
 
 [dependencies.proteus-traits]
 optional = true
-version = "2.0"
+workspace = true
```

Comment:
I would join both of these (`proteus-wasm` and `proteus-traits`) with the above table and make them inline, like we did with `uniffi` above and `proteus-traits` further below. I.e.
```
proteus-traits = { workspace = true, optional = true}
proteus-wasm =  { workspace = true, optional = true, features = ["hazmat"] }
```